crates/nassun/src/fetch/git.rs

```diff
@@ -223,3 +223,98 @@ impl PackageFetcher for GitFetcher {
         todo!()
     }
 }
+
+#[cfg(test)]
+mod test {
+    use std::{fs::File, io::Write, process};
+
+    use oro_client::OroClient;
+    use oro_package_spec::{GitInfo, PackageSpec};
+    use tempfile::tempdir;
+
+    use crate::fetch::PackageFetcher;
+
+    use super::GitFetcher;
+
+    fn setup_git_dir() -> miette::Result<tempfile::TempDir> {
+        let git_dir = tempdir().unwrap();
+
+        process::Command::new("git")
+            .args(["init", "--initial-branch", "main"])
+            .current_dir(&git_dir)
+            .status()
+            .expect("Could not init git test directory");
+
+        let mut package_file = File::create(&git_dir.path().join("package.json")).unwrap();
+        package_file
+            .write_all(
+                r#"{
+            "name": "oro-test",
+            "version": "1.0.0"
+        }"#
+                .as_bytes(),
+            )
+            .unwrap();
+        drop(package_file);
+
+        // commit the first version
+        process::Command::new("git")
+            .args(["add", "package.json"])
+            .current_dir(&git_dir)
+            .status()
+            .expect("Could not add package.json to git repo");
+        process::Command::new("git")
+            .args(["commit", "-m", "First version"])
```

Comment:
You should add `--no-gpg-sign` so that folks (like me) who have gpg signing set up don't get prompted for our passphrases when running the test suite :)

Reply:
Fixed.